bricksrc/equipment.py

```diff
@@ -126,57 +147,55 @@
             "HWS": {OWL.equivalentClass: BRICK["Hot_Water_System"], "tags": [TAG.HWS]},
         },
     },
-    "Steam_System": {"tags": [TAG.Steam, TAG.Equipment]},
+    "Water_Distribution": {
+        "tags": [TAG.Water, TAG.Distribution, TAG.Equipment],
+        SKOS.definition: Literal(
+            "Utilize a water distribution source to represent how water is distributed across multiple destinations (pipes)"
+        ),
+    },
+    "Steam_Distribution": {
+        "tags": [TAG.Steam, TAG.Distribution, TAG.Equipment],
+        SKOS.definition: Literal(
+            "Utilize a steam distribution source to represent how steam is distributed across multiple destinations"
+        ),
+    },
```

Comment:
Where do `Water Distribution` and `Steam Distribution` come from? These might have more to do with the System modeling than with the equipment

Reply:
Do you think we should remove them for now?

Reply:
Actually, let's keep for now, but we should consider what possible subclasses would be